src/eos/eosTypes.ts

```diff
@@ -81,10 +85,10 @@ export interface EosAction {
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
 export interface EosParams {}
 
-export interface EosWalletOtherData {
-  accountName: string
-  // @ts-expect-error
-  lastQueryActionSeq: { [string]: number }
-  // @ts-expect-error
-  highestTxHeight: { [string]: number }
-}
+export const asEosWalletOtherData = asObject({
+  accountName: asSafeOptional(asString, ''),
+  lastQueryActionSeq: asSafeOptional(asMap(asNumber), {}),
+  highestTxHeight: asSafeOptional(asMap(asNumber), {})
```

Comment:
Just replace `asMap` with `asObject`. When `asObject` receives a cleaner as its parameter (as opposed to an object), it works like `asMap` used to work. The `asMap` cleaner is deprecated, and will go away in the next breaking version.

Reply:
👍